packages/apps/crm/src/screens/client/ClientFormScreen.js

```diff
@@ -40,6 +40,20 @@ const ClientFormScreen = ({navigation}) => {
     [navigation],
   );
 
+  const createClientAPI = useCallback((objectState, dispatch) => {
+    dispatch(
+      createClient({
+        client: {
+          ...objectState,
+          isContact: false,
+          isCorporatePartner: false,
+          isCustomer: true,
+          isEmployee: false,
+        },
+      }),
+    );
```

Comment:
In our context, we should go back to ActiveCartScreen once client is created 
Add a props on the screen onCreate which should be executed after this dispatch and override the screen in sale to add the logic. onCreate should receive the id of the created client.